modules/ROOT/pages/provisioning-virtualbox.adoc

```diff
@@ -4,193 +4,144 @@ This guide shows how to provision new Fedora CoreOS (FCOS) nodes on the VirtualB
-
-We will rely on the VirtualBox-provided NAT gateway to access the hosts' loopback interface on `10.0.2.2`. Check the https://www.virtualbox.org/manual/UserManual.html#network_nat[VirtualBox User Manual] for more details.
+Fedora CoreOS is designed to be updated automatically, with different schedules per stream.
+Once you have picked the relevant stream, you can download the latest OVA:
 
 [source, bash]
 ----
-VM_NAME='fcos-node01'
-VM_DISK=$(realpath ~/VirtualBox\ VMs/${VM_NAME})/${VM_NAME}.vdi
-
-vboxmanage createvm --name "${VM_NAME}" --ostype Fedora_64 --register
-# Set hardware settings for the VM, including recommended graphics controller
-vboxmanage modifyvm "${VM_NAME}" --memory 4096 --cpus 2 --vram 20 --graphicscontroller vmsvga --rtcuseutc on
-# Add a storage controller for hard disks
-vboxmanage storagectl "${VM_NAME}" --name SATA --add sata --controller IntelAhci --portcount 30 --bootable on
+STREAM="stable"
+mkdir ova-templates
+coreos-installer download -s "${STREAM}" -p virtualbox -f ova -C ./ova-templates/
 ----
 
-WARNING: VirtualBox and by extension `vboxmanage` always expect absolute file paths when specifying locations.
+Alternatively, OVA images can be manually downloaded from the https://getfedora.org/coreos/download?tab=metal_virtualized[download page].
 
-Make sure to create a port forwarding rule that allows access to the guest's SSH Port with the https://www.virtualbox.org/manual/UserManual.html#networkingmodes[default NAT setting] using e.g. `localhost:2222` as target address:
-[source, bash]
-----
-vboxmanage modifyvm "${VM_NAME}" --natpf1 "guestssh,tcp,,2222,,22"
-----
+== Booting a new VM on VirtualBox
 
-The virtual machine is now ready and FCOS can be installed.
+You can set up a VirtualBox virtual machine through the GUI or via the https://www.virtualbox.org/manual/UserManual.html#vboxmanage[`vboxmanage` CLI]. This guide will use the CLI for setting up the VM.
 
-== Installing from live ISO
+=== Importing the OVA
 
-To install FCOS into a VirtualBox virtual machine using the live ISO, follow these steps:
+To import the OVA, use `VBoxManage import`:
 
-- Download the latest ISO image from the https://getfedora.org/coreos/download?tab=metal_virtualized&stream=stable[download page] or with podman (see https://coreos.github.io/coreos-installer/cmd/download/[documentation] for options):
-[source, bash]
+[source, bash, subs="attributes"]
 ----
-podman run --security-opt label=disable --pull=always --rm -v .:/data -w /data \
-    quay.io/coreos/coreos-installer:release download -f iso
+VM_NAME=my-instance
+VBoxManage import --vsys 0 --vmname "$VM_NAME" fedora-coreos-{stable-version}-virtualbox.x86_64.ova
```

Comment:
I'm hesitant to use `stable-version` here.. We have a few other pages that do it something like this:

```suggestion
FCOS_VERSION='...'
VM_NAME=my-instance
VBoxManage import --vsys 0 --vmname "$VM_NAME" "fedora-coreos-${FCOS_VERSION}-virtualbox.x86_64.ova"
```

Reply:
> I'm hesitant to use `stable-version` here

Hmm, why not?

Reply:
partially because we have 3 streams and could be using `testing` or `next` here instead.

Reply:
yes it is. Actually I wasn't aware of its use and prevalence in other pages.

oh hmm. I see - `{stable-version}` gets replaced on compile with https://github.com/coreos/fedora-coreos-docs/blob/0790bcddf55a3c3d2a1b109442879cb211ee7183/antora.yml#L11

I had missed that.

No reason to hold up this PR on that change now that I understand it a bit better. I do think that hardcoding in the version can lead to some confusion. Might be nice if a user had a slider or option to select the stream and then it would update the docs page.

Reply:
This sounds like a larger ongoing conversation. Going to resolve this and recommend revising once the larger conversation happens.

Reply:
@bgilbert RFE in https://github.com/coreos/fedora-coreos-docs/issues/395